internal/core/application/blockchain_listener.go

```diff
@@ -93,6 +99,7 @@ func newBlockchainListener(
 		pendingObservables:  make([]crawler.Observable, 0),
 		marketBaseAsset:     marketBaseAsset,
 		feeBalanceThreshold: feeBalanceThreshold,
+		network:             net,
```

Comment:
Except for trade and unspent epo, are other repos necessary if we only watch txs now?

Reply:
Yes. They're are all still used to retrieve info about a trade . MarketRepo is used to get the market account index given its quote asset, while VaultRepo is used to retrieve addresses info for some account (fee/market).